test/fixtures/message_split_to_header.eml

```diff
@@ -0,0 +1,32 @@
+DKIM-Signature: v=1; a=rsa-sha1; c=relaxed/simple; d=example.com; s=test;
+	t=1172620939; bh=ll/0h2aWgG+D3ewmE4Y3pY7Ukz8=; h=Received:Received:
+	 Received:From:To:Date:Subject:Message-ID; b=bj9kVUbnBYfe9sVzH9lT45
+	TFKO3eQnDbXLfgmgu/b5QgxcnhT9ojnV2IAM4KUO8+hOo5sDEu5Co/0GASH0vHpSV4P
+	377Iwew3FxvLpHsVbVKgXzoKD4QSbHRpWNxyL6LypaaqFa96YqjXuYXr0vpb88hticn
+	6I16//WThMz8fMU=
+Received: received data 0
+Received: received data 1
+Received: received data 2
+Received: received data 3 part 1
+	 data 3 part 2
+From: Murray S. Kucherawy <msk@sendmail.com>
+To:
+	Sendmail Test Address <sa-test@sendmail.net>
```

Comment:
The `'To:` header here is what's causing this message to fail.  This happens frequently with messages from yahoo.